updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc

```diff
@@ -0,0 +1,136 @@
+:_content-type: ASSEMBLY
+[id="updating-restricted-network-cluster-OSUS"]
+= Updating a restricted network cluster using the OpenShift Update Service
+include::_attributes/common-attributes.adoc[]
+:context: updating-restricted-network-cluster-OSUS
+
+toc::[]
+
+[id="update-restricted-network-cluster-update-service"]
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Understanding upgrade channels and releases]
+
+For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information.
```

Comment:
nit: "provides over-the-air updates" -> "provides over-the-air update recommendations", to make it clear that the update service is just information like "updates to A an B are recommended, while updates to C and D are supported, but possibly not recommended".  The actual updates (changing deployed component version running in the cluster) can be applied with or without that sort of advice.

Reply:
Implementing this feedback, and saving the rest of your comments for a future PR that focuses on content changes. Thank you!